api/app/GraphQL/Validators/CreateDevelopmentProgramInterestInputValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\GraphQL\Validators;
+
+use App\Enums\DevelopmentProgramParticipationStatus;
+use Database\Helpers\ApiErrorEnums;
+use Illuminate\Validation\Rule;
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class CreateDevelopmentProgramInterestInputValidator extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+        return [
+            // developmentProgramId validated in the Create/UpdateCommunityInterestInputValidator
```

Comment:
Unfortunately, it's a bit awkward that we can't validate this member of the input in its own validator.  We don't have the community_id at this level and having two validators for the same field doesn't seem to work in Laravel.